.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Product Description
+<!--
+Delete this section if the PR does not contain any visible changes.
+For non-invisible changes, describe the user-facing effects.
+-->
+
+## Technical Summary
+<!--
+Provide a link to the ticket or document which prompted this change.
+Describe the rationale and design decisions.
+-->
+
+## Safety Assurance
+
+### Safety story
+<!--
+Describe how you became confident in this change, such as local testing, why the
+change is inherently safe, and/or plans to limit the blast radius of a defect.
+
+In particular consider how existing data may be impacted by this change.
+-->
+
+### Automated test coverage
+<!-- Identify the related test coverage and the tests it would catch -->
+
+### QA Plan
+<!--
+- Describe QA plan that along with automated test coverages proves this PR is regression free.
+- Link to QA Ticket
+-->
+
+### Migrations
+<!--
+Delete this section if the PR does not contain any migrations.
+See also: https://commcare-hq.readthedocs.io/migrations_in_practice.html
```

Comment:
these docs don't really apply here

Reply:
Thanks @snopoke, I was a little conflicted on this myself. I looked around for "formplayer migration best practice" docs and couldn't find any and figured this doc was better than nothing (it offers _some_ generic migrations patterns).  Do you know if there's a migrations doc for formplayer that I can link in its place? If not I'll just remove this one without a replacement.

Reply:
There isn't an equivalent doc. FP also does very few migrations of any kind.